src/Raven.Studio/typescript/widgets/virtualGrid/textColumn.ts

```diff
@@ -0,0 +1,25 @@
+﻿import virtualColumn = require("widgets/virtualGrid/virtualColumn");
+
+/**
+ * Virtual grid column that renders text in its cells.
+ */
+class textColumn implements virtualColumn {
+    readonly cellClass = "text-cell";
+
+    constructor(
+        public dataMemberName: string,
+        public display: string,
+        public width: string) {
+    }
+
+    renderCell(item: Object, isSelected: boolean): string {
+        const cellValue = (item as any)[this.dataMemberName];
+        if (cellValue != null) {
```

Comment:
I have to agree with you.  javascript never stops amazing me. just like adding 2 empty arrays produce an empty string.